### PR TITLE
[Bug 1154236] Remove locale links from the l10n dashboard

### PR DESCRIPTION
**Pull request by MikkCZ**

### CODE REVIEW

File: kitsune/dashboards/jinja2/dashboards/localization.html [hunk 1]
```diff
@@ -65,18 +65,6 @@
...
       {{ product_choice_list(products, product, url('dashboards.localization')) }}
 
-      <div class="choice-list">
-        <label>{{ _('Locale:') }}</label>
-        <ul>
-          <li>
-            <a href="{{ url('dashboards.contributors') }}">{{ default_locale_name }}</a>
-          </li>
-          <li>
-            {{ current_locale_name }}
```
Comment (safwanrahman) [inline kitsune/dashboards/jinja2/dashboards/localization.html]: You should not remove the current locale name!

File: kitsune/dashboards/jinja2/dashboards/localization.html [hunk 2]
```diff
@@ -65,18 +65,6 @@
 
       {{ product_choice_list(products, product, url('dashboards.localization')) }}
 
-      <div class="choice-list">
-        <label>{{ _('Locale:') }}</label>
-        <ul>
-          <li>
-            <a href="{{ url('dashboards.contributors') }}">{{ default_locale_name }}</a>
```
Comment (safwanrahman) [inline kitsune/dashboards/jinja2/dashboards/localization.html]: Better to remove it from the views if the context is there also

File: kitsune/dashboards/jinja2/dashboards/localization.html (hunk 1 above)
Comment (MikkCZ) [inline kitsune/dashboards/jinja2/dashboards/localization.html]: Updated.

File: kitsune/dashboards/jinja2/dashboards/localization.html (hunk 2 above)
Comment (MikkCZ) [inline kitsune/dashboards/jinja2/dashboards/localization.html]: I am not sure what you mean. I haven't found anything related to this in views.

**Comment by MikkCZ** [comment]
https://bug1154236.bmoattachments.org/attachment.cgi?id=8743974

@safwanrahman @glogiotatidis r?

**Comment by glogiotatidis** [comment]
Code wise this is OK but I'm waiting on Vesper to r+ in the bug. Thanks @MikkCZ

**Comment by safwanrahman** [comment]
I need to take a look on it. will check it tonight

**Comment by glogiotatidis** [comment]
> I need to take a look on it. will check it tonight

Did you test this? Looks good?

**Comment by safwanrahman** [comment]
Yap. Looks Good. r+ Thanks @MikkCZ :+1:

**Comment by glogiotatidis** [comment]
Thnx!

**Pull request merged**